src/senaite/ast/upgrade/v01_01_000.zcml

```diff
@@ -0,0 +1,21 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
+    i18n_domain="senaite.ast">
+
+  <genericsetup:upgradeStep
+      title="SENAITE AST 1.1.0: Fix wrong 'NA/-' results from resistance tests"
+      description="Fix wrong 'NA/-' results from resistance tests"
+      source="1100"
+      destination="1101"
+      handler=".v01_01_000.fix_wrong_results_resistance"
+      profile="senaite.ast:default"/>
+
+  <genericsetup:upgradeStep
+      title="Upgrade to SENAITE AST 1.1.0"
+      source="1.0.0"
+      destination="1.1.0"
```

Comment:
make the destination `1101` or above the source `1.1.0`

Reply:
Yes, I've seen this.
But since you changed the version in `metadata.xml` from `1.1.0` to `1101` I was wondering if the above upgrade step is listed or not?

Reply:
I guess https://github.com/senaite/senaite.ast/pull/24/commits/0ad8a3a225f7eb39b5efe87bed3f9d8174c5339c is enough